.github/workflows/test.yaml

```diff
@@ -722,6 +724,55 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh ${{ github.job}} ${{ matrix.platform }}
 
+  data-proxy:
+    needs: start-time
+    timeout-minutes: 60 # can take longer if platforms are down, so better protect
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - cloudflare-workers
+          - vercel-edge-functions
+        clientEngine: ['dataproxy']
+        os: [ubuntu-latest] #, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    concurrency: ${{ github.run_id }}-platforms-serverless-${{ matrix.platform }}
```

Comment:
this should to be removed or adapted (`platforms-serverless` => `data-proxy`) for consistency